src/DssSpell.t.sol

```diff
@@ -220,11 +220,11 @@ contract DssSpellTest is DSTest, DSMath {
         // Test for spell-specific parameters
         //
         spellValues = SpellValues({
-            deployed_spell:                 address(0xd161bDFF491790d5E02b487f2346Feb07B3e7054),        // populate with deployed spell if deployed
+            deployed_spell:                 address(0),        // populate with deployed spell if deployed
             deployed_spell_created:         1621621450,        // use get-created-timestamp.sh if deployed
             previous_spell:                 address(0x8EFE2c82bD31B67fa262c0D364773629f6EA828A),        // supply if there is a need to test prior to its cast() function being called on-chain.
```

Comment:
@gbalabasquer why was the previous spell not updated?

Reply:
Because it was a spell that won't actually be published so it felt not good assuming to have that spell as executed when actually never will do. Well it might do if we end up publishing it this Friday, but in that case it will be considered as the next spell.